proxy/api/src/error.rs

```diff
@@ -23,6 +23,9 @@ pub enum Error {
     #[error("the current session is in use by `{0}`")]
     SessionInUse(coco::Urn),
 
+    #[error("no session in use")]
```

Comment:
There is already a `NoSession` error in `http::error::Routing::NoSession`, will replace.